app/src/app.tsx

```diff
@@ -37,7 +40,7 @@ const App = () => {
                 setSnappables(snappables);
             })
             .catch((e) => console.log(e));
-    }, [setLoggedIn, setUserProfile, setSnappables]);
+    }, [setSnappables]);
```

Comment:
This should probably include `GetSnappables` too, again see [here](https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects).